pennylane_lightning/lightning_qubit/lightning_qubit.toml

```diff
@@ -116,7 +117,7 @@ runtime_code_generation = false
 # If the device supports adjoint
 quantum_adjoint = true
 # If the device supports quantum control instructions natively
-quantum_control = false
+quantum_control = true
```

Comment:
@dime10 I know you suggested this. In the frontend, we do the following:

```python
        if QJITDevice._check_quantum_control(config):  # pragma: nocover
            QJITDevice.operations += ["QCtrl"]
```

I unconditionally added QCtrl, like so:

```python
        if QJITDevice._check_quantum_control(config):  # pragma: nocover
            QJITDevice.operations += ["QCtrl"]
        QJITDevice.operations += ["QCtrl"]
```

and I get this error:

```python
    def trace_quantum(self, ctx, device, trace, qrp) -> QRegPromise:
        raise NotImplementedError("QCtrl does not support JAX quantum tracing")  # pragma: no cover
```

Would you reconsider keeping false for a while longer? Or what do you suggest?

Reply:
This is a deficiency on Catalyst's part I think, for the moment we should always decompose regardless of what the device says, since we have no representation for Control in MLIR.

The update should still take place here since the device now has the support. We'll catch up once Sergei finishes his implementation.

Reply:
Ok, so we keep it `true` here, but Catalyst will disable it forcefully until it supports it. Alright?

Reply:
I'm good with that if @dime10 is also good with that.